mock/rectrack/index.md

```diff
@@ -0,0 +1,16 @@
+# Recommendations Track
+## Menu
+* [Rec Track Explained](#) (below)
+* [Recommendation Track Readiness Best Practices](#) (below)
+* [<- Back to **Home**](../index.md)
+
+## Content
+This page gives an understanding of the W3C "RecTrack" or Recommendations Track.
+
+### Rec Track Explained
+Take and modify content from <https://github.com/w3c/wg-effectiveness/blob/master/process.md#w3c-recommenation-technical-report-flow-the-recommendation-track>
+
+### Recommendation Track Readiness Best Practices
+Take and modify content from <XXXXXXX>
```

Comment:
Include best practice advice to accompany all proposals for normative changes with tests that can be contributed to test suites, since they will be needed to exit CR anyway, and they help clarify exactly what the spec text needs to say.

Reply:
Added this text to the doc so we know to add it.